### PR TITLE
NovaSeqXPlus integration

### PR DESCRIPTION
**Pull request by ssjunnebo**
Along with some minor changes to improve readability

### CODE REVIEW

**Review by aanil — APPROVED** [review]
👍 Nice job with the var names, looks so much better now.

File: flowcell_parser/classes.py [hunk 1]
```diff
@@ -42,64 +42,64 @@ def parse(self, demultiplexingDir='Demultiplexing'):
...
         if "M0" in instrument:
-            ss_path = os.path.join(self.path, 'SampleSheet_copy.csv')
+            samplesheet_path = os.path.join(self.path, 'SampleSheet_copy.csv')
         else:
-            ss_path = os.path.join(self.path, 'SampleSheet.csv')
-        rinfo_path = os.path.join(self.path, 'RunInfo.xml')
-        rpar_path = os.path.join(self.path, 'runParameters.xml')
+            samplesheet_path = os.path.join(self.path, 'SampleSheet.csv')
+        run_info_path = os.path.join(self.path, 'RunInfo.xml')
+        run_parameters_path = os.path.join(self.path, 'runParameters.xml')
```
Comment (kedhammar) [inline flowcell_parser/classes.py:49]: This file is called "RunParameters.xml" in the NSX I believe.

**Review by kedhammar — CHANGES_REQUESTED** [review]

File: flowcell_parser/classes.py (hunk 1 above)
Comment (alneberg) [reply flowcell_parser/classes.py:49]: Yes, but TACA creates a link named `runParameters.xml` that points to the one with capital R.

Comment (alneberg) [reply flowcell_parser/classes.py:49]: Given that TACA has been run on this of course...

**Review by kedhammar — APPROVED** [review]

**Review by alneberg — APPROVED** [review]
Very good!

**Pull request merged**